src/WorkflowGui/Resources/config/services.yml

```diff
@@ -2,18 +2,20 @@ imports:
     - { resource: 'services/installer.yml' }
 
 services:
-    Youwe\Pimcore\WorkflowGui\Resolver\ConfigFileResolver:
+    _defaults:
+        autowire: true
```

Comment:
We don’t use autowire in bundles

Reply:
Didn't knew that. In that case I'll just drop that specific change. Thanks for your fast review :)